dom.bs

```diff
@@ -1065,6 +1065,15 @@ seen from the definition above, an <a>event listener</a> is a more broad concept
 method, when invoked, must run these steps:
 
 <ol>
+ <li>
+  <p>If the global object is a {{ServiceWorkerGlobalScope}} object and its associated
```

Comment:
This should be context object's global object.

I guess doing it this way is okay. We don't need a generic solution yet. But once we do, you might have to update service workers again.